cmd/exercise/exercise.go

```diff
@@ -79,6 +81,29 @@ func (e *exercises) checkFramework(log common.Logger) {
 			e.rmap = common.PopulateRouteMap(common.AllRoutes)
 			e.rmap = servegin.PreMigrationFixups(e.rmap)
 		}
+	case "Go-Swagger":
+		if e.standalone {
+			serveswagger.Setup()
+			routes := common.Routes{}
```

Comment:
`serveswagger.Setup()` already sets up routes and limits input types. The routes are accessible as `serveswagger.SwaggerParams.Rulebar`. I would start with that, and remove the ones that cause crashes. There should be no need to do anything more with input types.